storage/reads/datatypes/storage_common.proto

```diff
@@ -16,6 +16,9 @@ service Storage {
   // Read performs a read operation using the given ReadRequest
   rpc Read (ReadRequest) returns (stream ReadResponse);
 
+  // ReadFilter performs a filter operation at storage
+  rpc ReadFilter (ReadFilterRequest) returns (stream ReadResponse);
```

Comment:
Presumably you are dropping `Range` (as in `ReadRangeFilter`) given a time range is assumed for all requests and therefore redundant

Reply:
Yes there's no reason to have a separate ReadRange RPC as everything is a filter from the perspective of storage. The planner will ensure a range is always included in a call to storage or else planning will fail.